docs/UserGuide.md

```diff
@@ -411,7 +417,6 @@ _Details coming soon ..._
 
 ## Command summary
 
-_italic_ - optional
 
 | Action         | Format, Examples                                                                                                                                                                                                                                                                          |
 | -------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- | --- | --- | --- | --- | --- |
```

Comment:
Command summary yet to be updated. 
Also for `sort` command, could you also help to remove `[remark]`? Thank you!